### PR TITLE
Update Artifact placeholders

### PR DESCRIPTION
**Pull request by SusannaV**

### CODE REVIEW

File: src/components/Artifact.js [hunk 1]
```diff
@@ -84,17 +96,23 @@ export default function Artifact(props) {
...
-              <Col sm><span className='mini-header new'>Events [CREATE EVENT BUTTON]</span></Col>
+              <Col sm><span className='mini-header'>Events <span className='new'>[CREATE EVENT BUTTON]</span></span></Col>
             </Row>
-            {<span className="new"><ul><li>Not part of an event</li><li>Link to event</li></ul></span>}
+            {artifactDetails.events.length<1 ? <p>No linked events</p> : <ul> {artifactDetails.events.map(e => <li>
+                  <Link onClick={() => props.changeTab('Events')} to={`/events/${e.id}`}>{e.name}</Link></li>)}
+                  </ul>
+            }
           <Row>
             <Col sm><span className='mini-header new'>GM Notes</span></Col>
```
Comment (5annaha) [inline src/components/Artifact.js:117]: Just add GM Notes ("gm_notes" can be found from ​/science​/artifact​/{id} request response) similarly as other data.

**Review by 5annaha — APPROVED** [review]
Can be merged after the addition of GM Notes data.
Good job on adding Not part of a plot/No entries!

**Review by nicou — APPROVED** [review]
Nice 👌 Left some comments about missing `key` props that might cause issues.

File: src/components/Artifact.js [hunk 2]
```diff
@@ -84,17 +96,23 @@ export default function Artifact(props) {
...
+            {artifactDetails.plots.length<1 ? <p>No linked plots</p> : <ul> {artifactDetails.plots.map(p => <li>
+                  <Link onClick={() => props.changeTab('Plots')} to={`/plots/${p.id}`}>{p.name}</Link></li>)}
+                  </ul>
+            }
           <Row>
-              <Col sm><span className='mini-header new'>Events [CREATE EVENT BUTTON]</span></Col>
+              <Col sm><span className='mini-header'>Events <span className='new'>[CREATE EVENT BUTTON]</span></span></Col>
             </Row>
-            {<span className="new"><ul><li>Not part of an event</li><li>Link to event</li></ul></span>}
+            {artifactDetails.events.length<1 ? <p>No linked events</p> : <ul> {artifactDetails.events.map(e => <li>
```
[the remark this answers is not in the document]
Comment (SusannaV) [reply src/components/Artifact.js]: Thanks for pointing these out! Fixed/added :)

**Pull request merged**